yabt/graph.py

```diff
@@ -132,33 +134,93 @@ def generate_all_targets(conf: Config):
             yield norm_rel_target(relpath(root, conf.project_root), '//')
 
 
+class SeedRef:
+    """Helper class for tracking references to targets"""
+
+    def __init__(self):
```

Comment:
Please add here a description of what this is for, and where else to update if you add members

Reply:
done